src/Storages/Kafka/StorageKafka.cpp

```diff
@@ -690,9 +694,17 @@ void registerStorageKafka(StorageFactory & factory)
             }
         }
 
+        // Get and check client id
+         String client_id = kafka_settings.kafka_client_id.value;
+        if (args_count >= 11)
```

Comment:
1) I don't think we need to support it as position parameter at all.
2) default like clickhouse-{hostnames}-{dbname}-{table_name}-{consumerno} is required.